aten/src/ATen/native/TensorAdvancedIndexingUtils.h

```diff
@@ -0,0 +1,93 @@
+/*
+The contents of this file are used in ATen/native/TensorAdvancedIndexing.cpp and ATen/native/quantized/TensorAdvancedIndexing.cpp
```

Comment:
nit: I'd remove this, it's easy for this to become outdated

Reply:
will do

Reply:
would be good to resolve before landing